policyengine_us/reforms/congress/tlaib/income_security_package/income_security_package.py

```diff
@@ -0,0 +1,412 @@
+
+        def formula(tax_unit, period, parameters):
+            p = parameters(period).gov.contrib.congress.tlaib.income_security_package
+
+            # If ECPA is in effect, federal refundable CTC is zero
+            if p.end_child_poverty_act.in_effect:
+                return 0
+
+            # Otherwise calculate normally
+            ctc = parameters(period).gov.irs.credits.ctc
+
+            maximum_amount = tax_unit("ctc_refundable_maximum", period)
+            total_ctc = tax_unit("ctc", period)
+
+            if ctc.refundable.fully_refundable:
+                reduction = tax_unit("ctc_phase_out", period)
+                reduced_max_amount = max_(0, maximum_amount - reduction)
+                return min_(reduced_max_amount, total_ctc)
+
+            maximum_refundable_ctc = min_(maximum_amount, total_ctc)
+
+            phase_in = tax_unit("ctc_phase_in", period)
+            limiting_tax = tax_unit("ctc_limiting_tax_liability", period)
+            ctc_capped_by_tax = min_(total_ctc, limiting_tax)
+            ctc_capped_by_increased_tax = min_(total_ctc, limiting_tax + phase_in)
+            amount_ctc_would_increase = (
+                ctc_capped_by_increased_tax - ctc_capped_by_tax
+            )
+            return min_(maximum_refundable_ctc, amount_ctc_would_increase)
+
+    # Neutralize federal non-refundable CTC when ECPA is in effect
+    class non_refundable_ctc(Variable):
+        value_type = float
+        entity = TaxUnit
+        definition_period = YEAR
+        label = "federal non-refundable child tax credit"
+        unit = USD
+        documentation = "Non-refundable portion of the Child Tax Credit"
+
+        def formula(tax_unit, period, parameters):
+            p = parameters(period).gov.contrib.congress.tlaib.income_security_package
+
+            # If ECPA is in effect, federal non-refundable CTC is zero
+            if p.end_child_poverty_act.in_effect:
+                return 0
+
+            # Otherwise calculate normally
+            ctc = tax_unit("ctc", period)
+            refundable = tax_unit("refundable_ctc", period)
+            return ctc - refundable
+
+    class reform(Reform):
+        def apply(self):
+            # Variables are always updated, but their formulas check in_effect
+            self.update_variable(baby_bonus)
+            self.update_variable(boost_payment)
+            self.update_variable(boost_tax)
+            self.update_variable(ecpa_child_benefit)
+            self.update_variable(ecpa_adult_dependent_credit)
+            self.update_variable(ecpa_filer_credit)
+            self.update_variable(household_benefits)
+            self.update_variable(spm_unit_benefits)
+            self.update_variable(income_tax)
+
+            # ECPA replaces federal EITC and CTC but preserves them for state references
+            self.update_variable(eitc)
+            self.update_variable(refundable_ctc)
+            self.update_variable(non_refundable_ctc)
```

Comment:
So basically we want the federal eitc value to be still calculated but affect the net income tree? 

If so we should be appending the `refundable` and `non_refundable` federal credit lists to return a non 0 value without affecting federal tax liability